### ISSUE TITLE
Search?

### ISSUE THREAD

**Issue by austkel** [joeyism/linkedin_scraper#92]
Is there any way to search using this?

**Comment by joeyism** [comment]
No, I haven't implemented search on it. If you're up to the task, you can do it and submit a PR
